src/cli/commands/global.js

```diff
@@ -20,6 +20,12 @@ import * as fs from '../../util/fs.js';
 const nativeFs = require('fs');
 
 class GlobalAdd extends Add {
+  constructor(...args) {
+    super(...args);
```

Comment:
I think this is not transpiling correctly in CI due to us using a deprecated [node 4 babel plugin](https://github.com/jbach/babel-preset-es2015-node4). I'll look into this. Otherwise we'd have to use named arguments or .apply